testsuites/syncgateway/functional/tests/test_xattrs.py

```diff
@@ -1921,7 +1931,7 @@ def update_sdk_docs(client, docs_to_update, prop_to_update, number_updates):
                 log_info('Updating: {} from SDK'.format(random_doc_id))
                 cur_cas = doc.cas
                 client.upsert(random_doc_id, doc_body, cas=cur_cas)
-            except KeyExistsError:
+            except CouchbaseException:
```

Comment:
CouchbaseException is generic one, it may not catch the right exception. Is there any equivalent error for KeyExistsError for 3.0?

Reply:
no , we do not have